webpack.config.js

```diff
@@ -0,0 +1,34 @@
+const path = require('path');
+
+module.exports = {
+  entry: './sauce/main.js',
+
+  output: {
+    filename: 'source/common/res/features/ynabToolkit.js'
+  },
+
+  resolve: {
+    modules: [path.resolve(__dirname, 'sauce'), 'node_modules']
+  },
+
+  module: {
+    rules: [{
+      test: /\.js$/,
+      exclude: /node_modules/,
+      enforce: 'pre',
+      use: [{
+        loader: 'eslint-loader'
+      }]
+    }, {
+      test: /\.js$/,
+      include: [
+        path.resolve(__dirname, 'sauce')
+      ],
+      use: [{
+        loader: 'babel-loader?presets[]=es2015'
```

Comment:
Could we consider using the `latest` preset? I don't see a reason to hold it back given that we've got Babel in here now.

Random thought.